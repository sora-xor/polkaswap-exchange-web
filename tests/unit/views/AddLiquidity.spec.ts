// import { shallowMount, createLocalVue } from '@vue/test-utils'
// import Vuex from 'vuex'

// import AddLiquidity from '@/views/AddLiquidity.vue'
// import { tokens } from '../../mocks/tokens'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// localVue.use(Vuex)
// SoramitsuElementsImport(localVue)

// describe('CreatePair.vue', () => {
//   let actions
//   let getters
//   let store

//   beforeEach(() => {
//     TranslationMock(AddLiquidity)

//     actions = {
//       setFirstToken: jest.fn(),
//       setSecondToken: jest.fn(),
//       setFirstTokenValue: jest.fn(),
//       setSecondTokenValue: jest.fn(),
//       addLiquidity: jest.fn()
//     }

//     getters = {
//       firstToken: () => tokens[0],
//       secondToken: () => tokens[1],
//       firstTokenValue: () => 12,
//       secondTokenValue: () => 12
//     }

//     store = new Vuex.Store({
//       modules: {
//         createPair: {
//           namespaced: true,
//           actions,
//           getters
//         }
//       }
//     })
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(AddLiquidity, { localVue, store })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('AddLiquidity test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
