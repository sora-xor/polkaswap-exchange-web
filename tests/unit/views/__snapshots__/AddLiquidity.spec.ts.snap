// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePair.vue should renders correctly 1`] = `
<div
  class="container"
>
  <!---->
   
  <s-form-stub
    class="el-form--actions"
    labelposition="right"
    validateonrulechange="true"
    value="[object Object]"
  >
    <div
      class="input-container"
    >
      <div
        class="input-line"
      >
        <div
          class="input-title"
        >
          
        </div>
         
        <!---->
      </div>
       
      <div
        class="input-line"
      >
        <s-form-item-stub
          prop=""
          showmessage="true"
        >
          <s-input-stub
            accept=""
            autocomplete="off"
            borderradius="small"
            class="s-input--token-value"
            disabled="true"
            form=""
            label=""
            name=""
            placeholder="0.00"
            prefix=""
            size="big"
            suffix=""
            tabindex=""
            type="text"
            value="0.0"
          />
        </s-form-item-stub>
         
        <s-button-stub
          borderradius="mini"
          class="el-button--empty-token"
          icon="chevron-bottom-rounded"
          nativetype="button"
          size="small"
          tabindex="0"
          tooltip=""
          tooltipplacement="top"
          type="tertiary"
        >
          
          
        
        </s-button-stub>
      </div>
    </div>
     
    <s-icon-stub
      class="icon-divider"
      name="plus-rounded"
      size="medium"
    />
     
    <div
      class="input-container"
    >
      <div
        class="input-line"
      >
        <div
          class="input-title"
        >
          <span>
            
          </span>
        </div>
         
        <!---->
      </div>
       
      <div
        class="input-line"
      >
        <s-form-item-stub
          prop=""
          showmessage="true"
        >
          <s-input-stub
            accept=""
            autocomplete="off"
            borderradius="small"
            class="s-input--token-value"
            disabled="true"
            form=""
            label=""
            name=""
            placeholder="0.00"
            prefix=""
            size="big"
            suffix=""
            tabindex=""
            type="text"
            value="0.0"
          />
        </s-form-item-stub>
         
        <s-button-stub
          borderradius="mini"
          class="el-button--empty-token"
          icon="chevron-bottom-rounded"
          nativetype="button"
          size="small"
          tabindex="0"
          tooltip=""
          tooltipplacement="top"
          type="tertiary"
        >
          
          
        
        </s-button-stub>
      </div>
    </div>
     
    <s-button-stub
      borderradius="small"
      disabled="true"
      icon=""
      nativetype="button"
      size="medium"
      tabindex="0"
      tooltip=""
      tooltipplacement="top"
      type="primary"
    >
      
        
      
    </s-button-stub>
  </s-form-stub>
   
  <!---->
   
  <!---->
   
  <!---->
   
  <!---->
   
  <!---->
   
  <!---->
</div>
`;
